app/views/arclight/requests/_aeon_external_request_endpoint.html.erb

```diff
@@ -0,0 +1,11 @@
+<% document ||= @document %>
+<% aeon_external_request ||= Arclight::Requests::AeonExternalRequest.new(document, show_presenter(document)) %>
+
+<%= form_tag aeon_external_request.url, method: :post, class: 'al-request-form' do |f| %>
+  <% aeon_external_request.form_mapping.each do |key, value| %>
```

Comment:
Blacklight has a helper method that might be handy -- [`render_hash_as_hidden_fields`](https://github.com/projectblacklight/blacklight/blob/8b9041544553ce0dd0d979f9f39f78d94b30b46d/app/helpers/blacklight/hash_as_hidden_fields_helper_behavior.rb)